### DISCUSSION TITLE
How do I connect points with already defined handles?

### DISCUSSION THREAD

**Discussion by yesjustryan** [fontforge/fontforge#4537] [Q&A]
Hi, I don't know if this is an appropriate place to ask this question, but..

I have curve points with bezier handles; how do I connect them with a curve that follows the handles?

I tried make line/make arc/merge/join, but nothing works.

I attached the screenshot.

pls help😢

![Screenshot 2020-12-17 075502](https://user-images.githubusercontent.com/76134148/102416957-218d7200-403e-11eb-8e2b-3d1d20a8badc.png)

**Comment by ctrlcctrlv** [comment]
I've had this problem before. I've put null points on the handles, then did make line, then made new handles and put them where the points were. You can also note down the values in Point Info. I know that's all annoying...if you're doing this a lot I think you'd need to PR something or script it.

**Comment by skef** [comment]
You're asking about the particular case where the handles already exist. I believe the following works:

1. Select exactly one of the two points to be connected. For example, the corner point.
2. Pick the tool for the point type of the *other* point from the palette. For example, the smooth point tool.
3. Click on the other point. 

If you accidentally have more than one point selected, or vary from these steps in another way, something else will happen.

**Comment by Unknow0059** [comment]
I think this should be marked as answered.
